brambling/templates/brambling/invite.html

```diff
@@ -4,25 +4,26 @@
 
 {% block title %}Accept invitation – {{ block.super }}{% endblock %}
 {% block main %}
+	{% comment %}
+	If the person is already logged in, the invitation is currently automatically accepted.
+	{% endcomment %}
 	{% if invite %}
-		<h2>Accept invitation from {{ invite.user.get_full_name }}</h2>
-		<ul class="nav nav-pills" role="tablist">
-			<li class="active"><a href="#login" role="tab" data-toggle="tab">Log In</a></li>
-			<li><a href="#signup" role="tab" data-toggle="tab">Sign Up</a></li>
-		</ul>
-		<div class="tab-content">
-			<form id="login" class="tab-pane fade in active" action="{% url 'login' %}?next={{ request.path }}" method="post">
+		{% if invited_person_exists %}
+			<h2>Log in to accept the invitation from {{ invite.user.get_full_name }}</h2>
+			<form id="login" action="{% url 'login' %}?next={{ request.path }}" method="post">
 				{% csrf_token %}
 				{% form login_form %}
 				<button class="btn btn-primary-light" type="submit">Log in</button>
 				<a class='btn btn-link' href="{% url 'password_reset' %}">Forgot your password?</a>
 			</form>
-			<form id="signup" class="tab-pane fade" action="{% url "brambling_signup" %}?next={{ request.path }}" method="post">
+		{% else %}
+			<h2>Create an account to accept the invitation from {{ invite.user.get_full_name }}</h2>
+			<form id="signup" action="{% url "brambling_signup" %}?next={{ request.path }}" method="post">
 				{% csrf_token %}
 				{% form signup_form using "brambling/forms/signup.html" %}
 				<button class="btn btn-primary-light">Sign Up</button>
```

Comment:
It *would* be nice if we could prevent people from logging in / signing up with other email addresses, since it wouldn't work (and then they'd be confused) but I think this should cover most cases as-is.